smart_campus/app/templates/app/login.html

```diff
@@ -0,0 +1,59 @@
+
+{% extends "app/base.html" %}
+{% load static %}
+{% block title %} Smart Campus Login {% endblock %}
+
+
+{% block stylesheet %}{% static "app/css/login.css" %}{% endblock %}
+{% block nav %}{% endblock %}
```

Comment:
Did you intentionally overwrite "block nav"?
Can you clarify why this is needed?

Reply:
Because I don't want navbar to show on login page